### PR TITLE
Move hardcoding text to lang file

### PR DESCRIPTION
**Pull request by TomoTsuyuki**

### CODE REVIEW

File: lang/en/local_edit_course_notification.php [hunk 1]
```diff
@@ -27,6 +27,10 @@
 
 $string['notificationstatement'] = '';
 $string['notificationtype'] = 'Information';
+$string['notificationtype:error'] = 'Error';
+$string['notificationtype:information'] = 'Information';
```
Comment (dmitriim) [inline lang/en/local_edit_course_notification.php:29]: is it duplicate of $string['notificationtype'] ? or they are different?

Comment (TomoTsuyuki) [reply lang/en/local_edit_course_notification.php:29]: Actually $string['notificationtype'] is for default value.

Comment (dmitriim) [reply lang/en/local_edit_course_notification.php:29]: then you are doing it wrong. It should be 'info'

File: settings.php [hunk 2]
```diff
@@ -39,10 +39,10 @@
             get_string('setting:notificationtype_desc', 'local_edit_course_notification'),
             get_string('notificationtype', 'local_edit_course_notification'),
```
Comment (dmitriim) [inline settings.php]: this should be \core\output\notification::NOTIFY_INFO

Comment (TomoTsuyuki) [reply settings.php]: Thanks for finding the bug.
I updated the code.

File: lang/en/local_edit_course_notification.php [hunk 3]
```diff
@@ -26,7 +26,11 @@
 defined('MOODLE_INTERNAL') || die();
 
 $string['notificationstatement'] = '';
-$string['notificationtype'] = 'Information';
+$string['notificationtype'] = \core\output\notification::NOTIFY_INFO;
```
Comment (dmitriim) [inline lang/en/local_edit_course_notification.php]: It doesn't make sense to have it as a lang string. Can you please remove this line and move \core\output\notification::NOTIFY_INFO to setting.php to be a default setting.

Comment (TomoTsuyuki) [reply lang/en/local_edit_course_notification.php]: I understand what you mean.
We don't need to use lang file if default values are for select, or empty.
I moved default value to settings.php

**Pull request merged**